androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -156,7 +158,8 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         mTwaLauncher.launch(twaBuilder,
                 mSplashScreenStrategy,
                 () -> mBrowserWasLaunched = true,
-                getFallbackStrategy());
+                getFallbackStrategy(),
+                getCustomHeaders());
```

Comment:
since the goal of these headers is to be attached to the intent, shouldn't be part of the intent builder ?
We could extends TrustedWebActivityIntentBuilder and add properties to it...

Reply:
`TrustedWebActivityIntentBuilder` is part of `androidx.browser` and we don't have access to modify it here. It would be nice to have it there, but I'd rather not be blocked by a change upstream.